tests/nodes/deployer/config.yml

```
@@ -23,7 +23,8 @@ rpc:
 
 cch:
   ignore_startup_failure: true
-  wrapped_btc_type_script_args: "0x32e555f3ff8e135cece1351a6a2971518392c1e30375c1e006ad0ce8eac07947"
+  # will be generated by udt-init
+  wrapped_btc_type_script: 0x55000000100000003000000031000000e1e354d6d643ad42724d40967e334984534e0367405c5ae42a9d7d63d77df419022000000032e555f3ff8e135cece1351a6a2971518392c1e30375c1e006ad0ce8eac07947
```

we may use ckb json type `Script` here, it's easy to read than a hex string:
```suggestion
  wrapped_btc_type_script:
      code_hash: 0xe1e354d6d643ad42724d40967e334984534e0367405c5ae42a9d7d63d77df419
      hash_type: Data1
      args: 0x....
```

---

wrapped_btc_type_script can also be specified with environment variable `CCH_WRAPPED_BTC_TYPE_SCRIPT`. It is not very easy to write nested structures with environment variable. Can we both do a hex decoding and a `ckb_json_types::Script` decoding? That makes both `0x55000000100000003000000031000000e1e354d6d643ad42724d40967e334984534e0367405c5ae42a9d7d63d77df419022000000032e555f3ff8e135cece1351a6a2971518392c1e30375c1e006ad0ce8eac07947` and `code_hash: 0xe1e354d6d643ad42724d40967e334984534e0367405c5ae42a9d7d63d77df419
      hash_type: Data1
      args: 0x....` legitimate.